code/controllers/subsystem/dcs.dm

```diff
@@ -1,6 +1,7 @@
 PROCESSING_SUBSYSTEM_DEF(dcs)
 	name = "Datum Component System"
 	flags = SS_NO_INIT
+	wait = 20
```

Comment:
is locking dcs processing into 2 second ticking a good idea?
i think it might be better to atleast pass something like time between fires/time after last fire and just have stuff scale to world.time instead of relying on this, like how i made projectiles. 
or actually, no, not world.time, but scaling the stuff in the actual element to the wait variable of the subsystem
my concern isn't like, keeping timing, but rather what changes in the future will do if we need this to tick at 1 second at some point.

Reply:
Yea, I'll just make the current components/elements process with SSobj instead, since they are all supposed to process at 2 seconds intervals, not 1, my bad. :stuck_out_tongue: